portafly/doc/ux_specs/product/applications/index.md

```diff
@@ -0,0 +1,31 @@
+# Applications index
+
+[Back to index](../../index.md)
+
+* [Mockups](https://marvelapp.com/ee82j74/screen/70440629)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4693)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4214)
+
+##### Page header
+* Create button
+  * Primary button
+
+##### Toolbar ([specs](../../global_components/toolbar.md))
+* Filters:
+  * Name > text input ([PF specs](https://www.patternfly.org/v4/documentation/react/components/inputgroup#with-dropdown))
+  * State > Dropdown menu with multi-select checklist ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#checkbox-input))
+     * "Live"
+     * "Pending"
+     * "Suspended"
+  * Account > text input ([PF specs](https://www.patternfly.org/v4/documentation/react/components/inputgroup#with-dropdown))
+  * Plan > Dropdown menu with multi-select checklist ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#checkbox-input))
+     * dropdown menu content depends on users application plans
+
+##### Table ([PF specs](https://www.patternfly.org/v4/documentation/react/components/table))
+* Table header:
+  * Name [sortable]
+  * State [sortable]
+  * Account [sortable]
+  * Plan [sortable]
+  * Created on [sortable]
+  * Traffic on [sortable]
```

Comment:
@cntlsn Does this refer to the latest activity with the application?

Reply:
If I'm not mistaken and according to Porta's applications page, you're assumption is correct @porueesq

Reply:
@porueesq @josemigallas  affirmative